### PR TITLE
chore: Update the express example

### PR DESCRIPTION
**Pull request by sp6370**
## Which problem is this PR solving?

- Updates to the express example #2528

## Short description of the changes

- Nukes the Zipkin usage and usage of Jaeger Otel Library
- Added a fix for the code compilation failing
- Update the otel dependencies to latest version
- Also fixed the deprecations for the semantic conventions

### CODE REVIEW

**Comment by sp6370** [comment]
@trentm

**Comment by sp6370** [comment]
@trentm Thanks for the awesome code review! 😄 

Definitely learned a lot from it!

LMK if anymore changes are needed.

File: examples/express/README.md [hunk 1]
```diff
@@ -17,44 +17,33 @@ shows key aspects of tracing such as
...
-
-## Run the Application
-
-### Zipkin
-
-Run the server:
-
-```sh
-npm run zipkin:server
+Start Jaeger in Docker for receiving tracing data (see [the Jaeger docs](https://www.jaegertracing.io/docs/2.0/getting-started/#in-docker) for more details about running Jaeger):
```
Comment (trentm) [inline examples/express/README.md:20]:
```suggestion
Start Jaeger in Docker for receiving tracing data (see [the Jaeger docs](https://www.jaegertracing.io/docs/2.0/getting-started/#in-docker) for more details about running Jaeger):

```

^^ need that blank line for `npm run lint:markdown` to pass.

**Review by trentm — COMMENTED** [review]
Just the one lint fix suggestion below. Otherwise, LGTM. Thanks!

**Review by trentm — APPROVED** [review]

**Pull request merged**